instance/models/openedx_instance.py

```diff
@@ -279,6 +279,11 @@ def get_load_balancer_configuration(self):
             # The active appserver doesn't have a public IP address.  This means that the server
```

Comment:
Yes, that's what's happening.  The only alternative to this approach is to cache the information in some form (e.g. by remembering the configuration snippets).

Note that reconfigurations of the load balancer aren't that frequent as you might think.  They currently happen about a dozen times per day.